libs/daohaus-connect/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "@daohaus/daohaus-connect",
```

Comment:
if we determine we want to change this to append `-feature` we'd need to update this too